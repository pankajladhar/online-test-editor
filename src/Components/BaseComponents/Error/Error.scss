.Error {
  color: #f44336;
  font-size: 14px;
  padding-top: 5px;
}