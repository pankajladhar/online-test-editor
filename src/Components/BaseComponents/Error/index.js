import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './Error.css';

class Error extends Component {
    render() {
        return (
            <div className="Error">
                {this.props.children}
            </div>
        );
    }
}

Error.propTypes = {

};

export default Error;