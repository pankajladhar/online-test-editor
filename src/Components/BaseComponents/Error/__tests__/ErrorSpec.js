import React, { Children } from 'react';
import renderer from 'react-test-renderer';
import Error from './../index';

describe('Error', () => {
    let errorProps = {}
    beforeEach(() => {
        errorProps = {
            children: <h1>Some Children</h1>
        }
    });

    it('renders correctly with mandatory params', () => {
        let tree = renderer.create(
            <Error {...errorProps} />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });


});