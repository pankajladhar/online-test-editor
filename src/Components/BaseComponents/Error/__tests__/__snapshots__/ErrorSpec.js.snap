// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly with mandatory params 1`] = `
<div
  className="Error"
>
  <h1>
    Some Children
  </h1>
</div>
`;
